packages-backend/express/package.json

```diff
@@ -42,7 +42,7 @@
     "jwks-rsa": "1.9.0"
   },
   "devDependencies": {
-    "jose": "1.28.0",
+    "jose": "2.0.2",
```

Comment:
Wait. No API changes we need to follow here?

Reply:
Not really, it's only used in this file: https://github.com/commercetools/merchant-center-application-kit/blob/master/packages-backend/express/src/middlewares/fixtures/jwt-token.ts

Reply:
Ok thanks. Just double checking.